### PR TITLE
Add dirs for RelEng and Licensing subprojects

### PR DESCRIPTION
**Pull request by justaugustus**
Placeholders and initial OWNERS for RelEng and Licensing subprojects
Labels will be in place once kubernetes/test-infra#10852 merges.
sigs.yaml update in kubernetes/community#3119

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/assign @calebamiles @tpepper @spiffxp @nikhita

### CODE REVIEW

**Comment by justaugustus** [comment]
/hold
(since we may bikeshed on the label names over here: https://github.com/kubernetes/test-infra/pull/10852#discussion_r249271002)

File: licensing/OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+approvers:
+  - sig-release-leads
+  - justaugustus
+  - nikhita
+  - swinslow
```
Comment (nikhita) [inline licensing/OWNERS]: @swinslow Can you [apply](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) to be a Kubernetes org member to have approval powers? :)

You can find more details [here](https://github.com/kubernetes/community/blob/master/community-membership.md).

**Comment by dims** [comment]
thanks @nikhita happy to help with this.

File: licensing/OWNERS (hunk 1 above)
Comment (swinslow) [reply licensing/OWNERS]: Will do -- thanks @nikhita  :)

File: release-engineering/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+approvers:
+  - sig-release-leads
+  - aleksandra-malinowska
+  - tpepper
+reviewers:
+  - branch-manager-role
+  - patch-release-manager-role
+labels:
+  - area/releng
```
Comment (spiffxp) [inline release-engineering/OWNERS]: `area/release-eng` ref: https://github.com/kubernetes/test-infra/pull/10852

File: licensing/OWNERS [hunk 3]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - sig-release-leads
```
Comment (spiffxp) [inline licensing/OWNERS]: Personal opinion: I don't think SIG Chairs should implicitly be subproject owners.  You have plenty of staff here

File: release-engineering/OWNERS [hunk 4]
```diff
@@ -0,0 +1,9 @@
+approvers:
+  - sig-release-leads
```
Comment (spiffxp) [inline release-engineering/OWNERS]: Personal opinion: I don't think SIG Chairs should implicitly be subproject owners.  You already have a chair here.

I would like to see someone from Google here, either @calebamiles or someone else who's been helping cut .deb's and .rpm's, from the perspective of our interest in debottlenecking.

Comment (tpepper) [reply release-engineering/OWNERS]: Do you mean "explicitly"?

I think sig-release-leads is redundant across these files' approvers.  There's still sig-release-leads at the next higher node's OWNERS file right?

Let's explicitly fully delegate unless there's an instance specific reason to not, but my initial feeling is that would be a sign of a prematurely formed and understaff subproject.

**Pull request merged**